cmake/Modules/FindOpenMP.cmake

```diff
@@ -101,7 +101,7 @@ function(_OPENMP_FLAG_CANDIDATES LANG)
       set(OMP_FLAG_Intel "-qopenmp")
     endif()
     set(OMP_FLAG_MIPSpro "-mp")
-    set(OMP_FLAG_MSVC "-openmp:experimental" "-openmp:experimental -I${__header_dir}" "-openmp" "-openmp -I${__header_dir}")
+    set(OMP_FLAG_MSVC "-openmp:experimental" "-openmp:experimental -I\"${__header_dir}\"" "-openmp" "-openmp -I\"${__header_dir}\"")
```

Comment:
This is for paths that has spaces in it, for example `C:\Program Files (x86)\Microsoft Visual Studio\2019\...`